### DISCUSSION TITLE
Module for python-broadlink

### DISCUSSION THREAD

**Discussion by malcb** [fhempy/fhempy#287] [Ideas]
May I suggest adding a module for https://github.com/mjg59/python-broadlink.

 Fhem does have a native broadlink module but that only handles 3 types where as python-broadlink has 52 types.  Hence, as with the eq3bt module, this would improve on functionality of the native module, even more significantly in this case.

Thanks for reading.

**Comment by fhempy** [comment]
I had a quick look at the API which looks quite good.

Which devices do you own for testing?

**Reply by malcb** [reply]
I have the BG smart socket (AHC).  I saw 4 in my local Aldi reduced to GBP7 ish and could not resist the bargain.  I've tried the python-broadlink  readme instructions and I can turn the unit on/off from the command line and read the state.
